packages/pilot/src/pages/Transactions/Search/Search.js

```diff
@@ -83,13 +92,26 @@ const normalizeQueryDatesToString = pipe(
   mergeAll
 )
 
-const stringToMoment = str => moment(str)
+const stringToMoment = str => moment(str, 'L')
+
+const normalizeDateTime = property => (date) => {
+  if (property === 'start') {
+    return date.startOf('day')
+  }
+
+  return date.endOf('day')
+}
 
 const normalizeStringToDate = property => pipe(
   prop(property),
-  unless(
+  ifElse(
     either(isNil, isEmpty),
-    pipe(stringToMoment, objOf(property))
+    () => ({ end: null, start: null }),
```

Comment:
No futuro podemos trocar isso por
```suggestion
    always({ end: null, start: null }),
```

Reply:
Boa, resolvido!